commander/src/bootstrapping/templates/lisk-template-ts/templates/init_plugin/test/unit/plugin.spec.ts

```diff
@@ -0,0 +1,24 @@
+// import * as plugins from '../../src/plugin';
+
+describe('<%= className %>Plugin', () => {
```

Comment:
```suggestion
describe('<%= className %>', () => {
```
class name already contains Plugin suffix